modules/ROOT/pages/upgrade-service-mesh.adoc

```diff
@@ -0,0 +1,60 @@
+= Upgrade Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, upgrading
+
+
+Anypoint Service Mesh v1.1 includes new features and architecture enhancements, such as bypassing the Istio Mixer when sending requests from clients to Anypoint Platform. This change caused an Anypoint Service Mesh architecture updates, and therefore, introduces a downtime for the Anypoint Service Mesh upgrade to v1.1.
+
+Anypoint Service Mesh v1.1 includes new features and architecture enhancements, such as stopping using Istio Mixer. Because of this change, upgrading from Anypoint Service Mesh v1.0 to v1.1 introduces downtime during the c
+
+== Prerequisites
+
+Before you upgrade, ensure that you meet the following requirements:
+
+. Ensure that the current Istio version in your environment is at least 1.6.0 (control plane and data plane):
+[source,text,linenum]
+----
+$ istioctl version
+client version: 1.6.0
+control plane version: 1.6.0
+data plane version: 1.6.0 (5 proxies)
+----
+. Download the latest Anypoint Service Mesh command line interface (CLI) version:
++
+`curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > asmctl && chmod +x asmctl`
+. Verify that you have the correct Anypoint Service Mesh and command line interface (CLI) versions  installed in your environment:
++
+`$ asmctl version`
++
+The CLI version must be v1.1.x and the Anypoint Service Mesh version must be v1.0.x.
+
+== Upgrade to Anypoint Service Mesh v1.1
+
+When you upgrade from Anypoint Service Mesh v1.1, the process first creates a backup of the necessary files. Any request to protected services is denied and the Anypoint Service Mesh downtime begins. The adapters, APIs, and bindings are then upgraded, and finally requests to protected services are resumed, thereby ending the Anypoint Service Mesh downtime. 
```

Comment:
Why does this introduction to user steps summarize the whole process? 
This sounds more like text that should be used to introduce the entire topic set, not the user steps.
What prompts the creation of the backup?
Why would there be requests to protected services; does the downtime automatically start, and what prompts that?
What prompts the upgrade of the adapters, APIs, and bindings: the one command documented here?
What prompts resumption of services/ending downtime?

Consider omitting this as an intro, and begin with something like...

Upgrade to Anypoint Service Mesh v.1.1:

`$ asmctl upgrade`

This single command initiates a process comprising the following actions:

. Protected service files are backed up.
. Requests for services are denied as downtime begins.
. Adapters, APIs, and bindings are upgraded.
. Downtime ends and services resume.

Reply:
Moved the para to the intro section. Done! Thanks!